buildSrc/src/main/kotlin/deps.kt

```diff
@@ -71,6 +71,9 @@ object Dependencies {
   const val kotlinReflect = "org.jetbrains.kotlin:kotlin-reflect:${Versions.kotlin}"
   const val kotlinStdlibOsgi = "org.jetbrains.kotlin:kotlin-osgi-bundle:${Versions.kotlin}"
   const val kotlinJunit5 = "org.jetbrains.kotlin:kotlin-test-junit5:${Versions.kotlin}"
+  const val kotlinTest = "org.jetbrains.kotlin:kotlin-test-common:${Versions.kotlin}"
```

Comment:
You don't need all these dependencies anymore. Just kotlin-test on commonTest.

Reply:
I want that to be true but it didn't work for me on my first attempt. I'll try it again.

Reply:
It doesn’t work as-is. I might need to upgrade things? Later.
```
> Task :okhttp:compileTestKotlinJvm FAILED
e: /Users/jwilson/Projects/okhttp/okhttp/src/jvmTest/java/okhttp3/MediaTypeGetTest.kt: (18, 20): Unresolved reference: assertEquals
e: /Users/jwilson/Projects/okhttp/okhttp/src/jvmTest/java/okhttp3/MediaTypeGetTest.kt: (19, 20): Unresolved reference: assertFailsWith
e: /Users/jwilson/Projects/okhttp/okhttp/src/jvmTest/java/okhttp3/MediaTypeTest.kt: (21, 20): Unresolved reference: Test
```